.github/workflows/tests.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     strategy:
       matrix:
         python-version: ['3.9', '3.12']
-        os: ['ubuntu-latest', 'windows-latest']
+        os: ['ubuntu-latest', 'macos-latest', 'windows-latest']
```

Comment:
I'm not sure this tells us any additional information beyond running on ubuntu.

Reply:
Yeah, that's probably true. Maybe we should also try the `commadore64-latest` runner  💻